src/components/InjectedComponents/PostDummy.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react'
+import { usePostInfoDetails } from '../DataSource/usePostInfo'
+import { DefaultTypedMessageRenderer } from './TypedMessageRenderer'
+import { PluginUI } from '../../plugins/plugin'
+import { makeTypedMessageCompound, isTypedMessageSuspended, isTypedMessageKnown } from '../../protocols/typed-message'
+import { useValueRef } from '../../utils/hooks/useValueRef'
+import { currentWholePostVisibilitySettings, WholePostVisibility } from '../../settings/settings'
+
+export interface PostDummyProps {
```

Comment:
I request to remove all usages of "Dummy", it seems like very un-informal.

Reply:
It's like test dummies which means replace the original one to do sth.